projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -293,14 +293,9 @@ public static function enqueue_scripts() {
 				),
 				'isStatsModuleActive'    => $modules->is_active( 'stats' ),
 				'canUserViewStats'       => current_user_can( 'manage_options' ) || current_user_can( 'view_stats' ),
-				'isCommercial'           => self::is_commercial_site(),
```

Comment:
This value is not being used anywhere on the frontend. The `is_commercial_site` is being used in the Stats product class, but the function that uses it is being async loaded with the rest of the async product data, so there's no need to remove this entirely and call it on the frontend